mcserver/utils/misc.py

```diff
@@ -65,7 +66,15 @@ def open_local(filename: str):
     dir_name = dirname(inspect.stack()[1].filename)
     return open(join(dir_name, filename))
 
-
+def read_favicon() -> str:
```

Comment:
This should be a Union[str, bool] now that we return False.

Reply:
True

Reply:
Better to return `None` and make it Optional[str]